### DISCUSSION TITLE
Loss sudden increase

### DISCUSSION THREAD

**Discussion by SmaugPool** [SWivid/F5-TTS#528] [Q&A]
What happened? 
Does this mean that the model has denegerated and would the best model be the one just before the increase in this case?
 
 Inference output produces empty sounds with the last models.

Note that I used the default learning rate of 0.000075 and 100000 epochs. Would other values have allowed to train longer?
![image](https://github.com/user-attachments/assets/a0bb5d20-7bd1-44c2-aa37-1762e824af4e)

**Comment by sngraam** [comment]
may your training data error!

**Reply by SmaugPool** [reply]
Not sure to understand what you mean. The training went fine during about a day. The loss increase came after.

**Reply by omid77922** [reply]
> Not sure to understand what you mean. The training went fine during about a day. The loss increase came after.

**Reply by omid77922** [reply]
> Not sure to understand what you mean. The training went fine during about a day. The loss increase came after.

**Comment by isolveit-aps** [comment]
Can I ask how you get the graph? Which folder do you point tensorboard to?

**Reply by SmaugPool** [reply]
> Can I ask how you get the graph? Which folder do you point tensorboard to?

Choose `tensorboard` logger, install it with `pip install tensorboard`, then start it with:
```
tensorboard --logdir runs/F5TTS_Base/
```
Then connect to the web interface by clicking the displayed URL.

**Reply by isolveit-aps** [reply]
Thank you @SmaugPool. Could it be that the sudden loss is because all the projects are using the same runs folder, so the sudden loss is for a different project than the previous one?

**Reply by SmaugPool** [reply]
I don't think so. I was training only one model, and after the loss increase, the model doesn't work anymore (no sound output), but it works before. I wonder if I could have been able to train longer with a lower learning rate (1e-5 for example instead of 7.5e-5).

**Comment by sarpba** [comment]
I also had the problem when I taught from scratch, but the quality of the training data was lower. It probably also depends on the data quality.
